### DISCUSSION TITLE
进程守护的重启功能

### DISCUSSION THREAD

**Discussion by maxbad** [1Panel-dev/1Panel#6066] [Q&A]
### 联系方式

maxbad@vip.qq.com

### 1Panel 版本

v1.10.10-lts

### 问题描述

进程守护的进程需要更新程序的时候, 如果直接在界面点击重启,程序是不能启动的 不知道什么原因, 需要等一分钟左右在点才能重启成功, 或者先点击停止,在点击启动, 也不能启动. 这不知道什么问题.,..

我现在的办法是: 先替换守护进程文件, 然后看一下当前进程PID,再去进程管理里面结束当前进程. 这样更新程序的

### 重现步骤

1,创建一个守护进程,并且启动
2,文件列表更新守护进程文件
3,进程守护界面,重启守护进程

**Comment by wanghe-fit2cloud** [comment]
感谢反馈，我们先在本地环境复现一下该问题。

**Comment by zhengkunwang223** [comment]
您好 本地环境没有复现 能否提供更多信息 比如进程是什么类型 进程数量多少
